src/Components/test/testassets/BasicTestApp/InteropComponent.razor

```diff
@@ -175,6 +176,41 @@
             ReturnValues["roundTripByteArrayWrapperObjectFromDotNet"] = ((IJSInProcessRuntime)JSRuntime).Invoke<ByteArrayInterop.ByteArrayWrapper>("roundTripByteArrayWrapperObject", byteArrayWrapperObject).ToString();
         }
 
+        // Streaming interop tests need to be run only for Blazor server
```

Comment:
Could you clarify why this is the case? Aren't the streaming APIs supposed to work in exactly the same way on WebAssembly?

Reply:
Given we don't put any restrictions in WASM byte array interop (unlike blazor server where we have max byte array size set by the max SignalR message size, less overhead), I didn't think we wanted to support streaming in WASM.

If we do want to support streaming in WASM, chunk size would be an open question. In the current blazor server implementation we determine chunk size based on the max SignalR message size. For WASM we have a lot more flexibility, we could theoretically "stream" the entire byte array in a single chunk, or just utilize larger chunks for the interop.

Thoughts?

Reply:
We nearly always try to make the same APIs work for both WASM and server. Otherwise people can’t write portable component libraries without complicated per-platform logic. 

Additionally even though on WASM you could send a 10MB buffer in one go, for many use cases you wouldn’t want to allocate such a large object on the .NET heap. 

So I do think we will want to make streaming work on WASM and WebView too. Can be a separate PR. Does that seem reasonable?

Reply:
Sounds good, will handle that in a separate PR so it doesn't block this.